helm/flannel-operator-chart/values.yaml

```diff
@@ -1,6 +1,30 @@
-clusterRoleBindingName: flannel-operator
-clusterRoleBindingNamePSP: flannel-operator-psp
-clusterRoleName: flannel-operator
-clusterRoleNamePSP: flannel-operator-psp
-namespace: giantswarm
-pspName: flannel-operator-psp
+image:
+  registry: "quay.io"
+  repository: "giantswarm/flannel-operator"
+  tag: "[[ .SHA ]]"
+pod:
+  user:
+    id: 0
```

Comment:
Why can't we change the owner of certs in node to group `giantswarm` so that we can get access as group `1000`

Reply:
that would definitely be the best method - where would this be applied?

Reply:
https://github.com/giantswarm/giantnetes-terraform
https://github.com/giantswarm/hive

Reply:
@corest sure, but that would be outside of the scope of this PR - it does need changing though